src/components/Gallery.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { GalleryPhotoDetails } from "../api/nasaApi";
+import "./Gallery.scss";
+import "../styles/_variables.scss";
+import Modal from "./Modal";
+interface GalleryProps {
+  galleryPhotos: GalleryPhotoDetails[];
+}
+
+const Gallery = ({ galleryPhotos }: GalleryProps) => {
+  const [selectedPhotoDetails, setSelectedPhotoDetails] =
+    useState<GalleryPhotoDetails>();
+
+  return (
+    <div className="Gallery">
```

Comment:
As above, change to match casing
```suggestion
    <div className="gallery">
```